pixart.js

```diff
@@ -1 +1,21 @@
+var brushBox = document.querySelector(".brush");
+var inputBox = document.innerHTML.querySelector("#color-field").value;
+var colorButton = document.querySelector("#set-color")/
 
+function setColor() {
+  document.body.style.background = "#1B4370";
+  document.body.style.background = ('inputBox').value
```

Comment:
you're really close here, but remember the difference between a *STRING* which is in quotes (single or double), vs a *VARIABLE* which is not in quotes, and is a stand-in for the value that it holds. Here you'd want to use the *variable* `inputBox`, which would be the actual HTML element you tried to define on line 2 (so close!).